### ISSUE TITLE
Messed up text

### ISSUE THREAD

**Issue by Shadowedcross** [ThibautSF/ParadoxosModManagerRework#31]
The text is complete gibberish and persists like this through reinstalls. 
![java_2020-04-15_19-35-28](https://user-images.githubusercontent.com/16232228/79375423-84c78180-7f50-11ea-9738-447b6bfe2140.png)

**Comment by Shadowedcross** [comment]
So I looked at the ASCII character set and comparing what I type vs what I get, most give me a character that's two places ahead so instead of 'a' I get 'c' or '{' instead of 'y'. Some characters work fine though, such as the exclamation mark or the dollar sign.

**Comment by ThibautSF** [comment]
This is really strange! 

All is compilated and text encoded in UTF8, and even if it's an encoding error, it's don't look like the standard errors between iso/utf8.

Do you use any specific language for Windows?

**Comment by Shadowedcross** [comment]
I just use the default which for me is English (United Kingdom)

**Comment by Shadowedcross** [comment]
The text that's in a different font seems to work just fine 
![java_2020-04-16_13-17-28](https://user-images.githubusercontent.com/16232228/79455461-d0c40600-7fe4-11ea-91c7-98ec4a0acdc4.png)

**Comment by ThibautSF** [comment]
So... One text is working (window title too in fact)

All text of PMM uses default system font, "Your mods ..." uses any serif family font

**Comment by Shadowedcross** [comment]
I don't believe I did, and just in-case I had, I restored default font settings but it's still broken.

**Comment by ThibautSF** [comment]
(Again I forgot that some keyword close issues...)

Can you try to run the new Indev build? 
https://github.com/ThibautSF/ParadoxosModManagerRework/releases/tag/indev

I made a basic fix which will need to take more time to improve it but I want to know if it's working.
The fix made all text to use a sans-serif family font ("Your mods..." will still be serif)

Could you also send back the `DebugLog.txt` file which will be generated in the root folder each time you start the app.

**Comment by Shadowedcross** [comment]
The indev build works 
![java_2020-04-17_12-09-53](https://user-images.githubusercontent.com/16232228/79563285-683c5e00-80a4-11ea-8ec6-ce81dd74fc00.png)

Here's the DebugLog 
[DebugLog.txt](https://github.com/ThibautSF/ParadoxosModManagerRework/files/4492525/DebugLog.txt)

**Comment by ThibautSF** [comment]
So there is something strange with the default font.

In the debug this line is good
`Font[name=System Regular, family=System, style=Regular, size=12.0]`

Can you give as many details as you think about your system?
What I can think: OS name, OS version, your desktop environment (in case of Linux systems) and its version...

Will try to see if this bug already occurs, or if it was already reported to openjfx (and in the other case report it myself)

**Comment by Shadowedcross** [comment]
Windows 10 Home Edition, Version 1903 (Build 18362.778)

**Comment by ThibautSF** [comment]
Windows 10? Your window has a different style than normal.
Is this a custom theme installed?

**Comment by Shadowedcross** [comment]
I'm using a custom theme yes, the text is still messed up if I switch back to the default theme though.

**Comment by ThibautSF** [comment]
Is it a theme customized by you, or a theme available distributed by someone (or a company)?

To test that on a Virtual Machine if it's the origin of the problem ^^

**Comment by Shadowedcross** [comment]
[This](https://www.deviantart.com/scope10/art/Penumbra-10-Windows-10-visual-style-568740374) is the theme I'm using.
